projects/Mallard/src/App.tsx

```diff
@@ -191,13 +193,16 @@ export default class App extends React.Component<{}, {}> {
                                     backgroundColor="#041f4a"
                                 />
                                 <View style={styles.appContainer}>
-                                    <RootNavigator
+                                    {/* <RootNavigator
```

Comment:
Old navigation entry point - the hope is everything from this point down will be deleted